src/bootstrap/native.rs

```diff
@@ -522,17 +544,22 @@ fn configure_cmake(
     if builder.config.llvm_clang_cl.is_some() {
         cflags.push_str(&format!(" --target={}", target))
     }
+    if let Ok(flags) = env::var("CFLAGS") {
```

Comment:
nit: env::var_os to avoid UTF-8 validity checking.

Reply:
So, I didn't make this `var_os` because `cflags` and `cxxflags` are both `String`s already. I suppose I can change them to `OsString` 👍